### ISSUE TITLE
txng.collection(col).insert(row, return_new=True) does not return a new row

### ISSUE THREAD

**Issue by zopyx** [joowani/python-arango#59]
`return_new=True` work perfectly as announced with normal insertion into a collection.
However an insertion using an transaction manager does not return anything like:

```
row = txn.collection('mycol').insert(data, return_new=True)
# row always None

```
Tested with 3.11.0

**Comment by zopyx** [comment]
ok, this behavior is documented.

Let me rephrase: how can you obtain the _key of all inserted documents during a transaction?

**Comment by dancollins34** [comment]
So it looks like you might be able to do something like this in your query:

```python
txn = db.transaction(write=col_name)
result = txn.execute(
    command='''
    function (params) {{
        var db = require('internal').db;
        var a = db.{col}.save({{ '_key': '1', 'val': params.one }});
        var b = db.{col}.save({{ '_key': '2', 'val': params.two }});
        return [a,b];
    }}'''.format(col=col_name),
    params={'one': 3, 'two': 4}
)
```

And that should return the meta information including the _key on each object.

**Comment by dancollins34** [comment]
It looks like this could be handled in a relatively straightforward manner by maintaining a javascript map of outputs to return.  However, I think the following code should be moved to transaction.py handle_request (from 104, collections/standard.py insert)

```
if self._conn.type != 'transaction':
            command = None
        else:
            command = 'db.{}.insert({},{})'.format(
                self._name,
                dumps(document),
                dumps(params)
            )
```

I'll probably submit a request to do this tonight.

**Comment by dancollins34** [comment]
@joowani take a look, but I think I addressed this in #pr66

**Comment by joowani** [comment]
@zopyx In python-arango version [4.0.0](https://github.com/joowani/python-arango/releases/tag/4.0.0), transactions return job objects allowing result retrieval after commit. You can read more about it [here](http://python-driver-for-arangodb.readthedocs.io/en/master/transaction.html). Please note that 4.0.0 is not backward compatible and you may need to significantly change your code. Cheers.
